### PR TITLE
Add libsecret

### PR DESCRIPTION
**Pull request by Lctrs**
Same reasons/motivations as for openjpeg

See https://github.com/search?q=org%3Aflathub+libsecret&type=Code

### CODE REVIEW

**Comment by TingPing** [comment]
Seems reasonable, do you plan on updating the places that use it?

File: libsecret/libsecret.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "-Dvapi=false",
+    "-Dgtk_doc=false"
+  ],
+  "cleanup": [
+    "/bin",
+    "/include",
+    "/lib/pkgconfig",
+    "/share/gir-1.0",
+    "/share/man",
+    "*.la"
```
Comment (TingPing) [inline libsecret/libsecret.json]:
```suggestion
```

**Comment by Lctrs** [comment]
> Seems reasonable, do you plan on updating the places that use it?

Yes.

File: libsecret/libsecret.json (hunk 1 above)
Comment (Lctrs) [reply libsecret/libsecret.json]: Done.

**Pull request merged**